internal/controller/operatorconfigmap_controller.go

```diff
@@ -314,7 +316,57 @@ func (c *OperatorConfigMapReconciler) Reconcile(ctx context.Context, req ctrl.Re
 	return ctrl.Result{}, nil
 }
 
+func (c *OperatorConfigMapReconciler) errorOnRookOwnedCsi() error {
+	rookManagedCsiObjects := []struct {
+		gvk  schema.GroupVersionKind
+		name string
+	}{
+		{
+			gvk:  appsv1.SchemeGroupVersion.WithKind("Deployment"),
+			name: "csi-rbdplugin-provisioner",
+		},
+		{
+			gvk:  appsv1.SchemeGroupVersion.WithKind("Deployment"),
+			name: "csi-cephfsplugin-provisioner",
+		},
+		{
+			gvk:  appsv1.SchemeGroupVersion.WithKind("Deployment"),
+			name: "csi-nfsplugin-provisioner",
+		},
```

Comment:
What is the reason to check the provisioners?

Reply:
We don't want half from rook and another half from csi-op

Reply:
Rook also creates the services for csi driver, IMO we are adding extra code that is not required for this problem.

Reply:
Lets followup to remove the deployments